.github/workflows/dev-workflow.yml

```diff
@@ -191,32 +191,57 @@ jobs:
           npm i -g aws-cdk
           make deploy
 
-  nextjs-web:
-    runs-on: ubuntu-latest
```

Comment:
@Jamesunekwuojo 
Workflows need to be changed when necessary, but this seems like an unnecessary change.